.github/dependabot.yml

```diff
@@ -5,7 +5,31 @@ updates:
     directory: '/'
     schedule:
       interval: 'monthly'
-    ignore:
```

Comment:
Is this safe to remove? The comment seems to indicate that it's there for some reason.

Reply:
We're not using Husky anymore. It's nowhere to be found in our lockfile.